default/scripting/species/common/influence.macros

```diff
@@ -1,4 +1,24 @@
 
+// (Owned non-exobot colonies + 0.25*(owned exobots and outposts)) / total planets in galaxy
+PERCENTAGE_OWNED_PLANETS
+'''((Statistic Count condition = And [
+                        Planet
+                        OwnedBy empire = Source.Owner
+                        Species
+                        Not Species name = "SP_EXOBOT"
+                    ]
+                    + ((NamedReal name = "OUTPOST_RELATIVE_ADMIN_COUNT" value = 0.25)
+                    * Statistic Count condition = And [
+                        Planet
+                        OwnedBy empire = Source.Owner
+                        Or [
+                            Not Species
+                            Species name = "SP_EXOBOT"
+                        ]
+                    ]))
+                    / (Statistic Count condition = Planet))'''
```

Comment:
I would rather have the dednominator depend on the number of visible planets or planets that have ever been seen by the empire, not the number that it has no way of knowing about.

Reply:
> have the dednominator depend on the number of visible planets or planets that have ever been seen

That would break the equation, demanding all planets set to influence and still go broke, because the equation then would assume most of the galaxy has been colonized and that the tech/policies are end game so lots of IP.

Reply:
> That would break the equation

OK, but you could fix it. I quite dislike the upkeep costs being dependent on the size of galaxy that an empire has no information about, and it would also give an extra benefit for exploration.

Reply:
I don't know if it can be fixed. I'll try.

But I quite dislike any form of upkeep that doesn't take into account galaxy size, for the reasons explained in several places (I could repeat them), and because I've been playing with current formula and I know it's annoying and impossible to balance for wide ranges of galaxy sizes.

Geoff, why do you dislike the upkeep based on galaxy size? The number of systems in the universe is public information, for human players and AIs.
Plus knowing the exact number of planets in the galaxy gives the player no advantage at all. I really don't understand your opposition.

Also, exploration is not a neglected department, it doesn't need extra motivation.